src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/ExtendedQueryTagEntryValidator.cs

```diff
@@ -60,6 +65,12 @@ public void ValidateExtendedQueryTags(IEnumerable<ExtendedQueryTagEntry> extende
                 throw new ExtendedQueryTagEntryValidationException(DicomCoreResource.MissingExtendedQueryTag);
             }
 
+            if (extendedQueryTagEntries.Count() > _maxBulkAddSize)
```

Comment:
I thought this was to limit max allowed per service, instead of each request. Why this limit?

Reply:
* will update the user story
* This need updates database storage procedure, thinking to combine the change with other coming updates like reindexing.